### ISSUE TITLE
Load 3rd party script failed

### ISSUE THREAD

**Issue by rexrainbow** [photonstorm/phaser#4572]
## Version

* Phaser Version: 3.17
* Operating system: win10
* Browser: chrome, and firefox

## Description

I try to load youtube iframe api from url: https://www.youtube.com/iframe_api
[Here](https://codepen.io/rexrainbow/pen/XwOVrG) is a worked test sample code, copied from official youtube api webpage.
[Here](https://codepen.io/rexrainbow/pen/jodagx) is a test sample code of using phaser3's script file loader -
```javasrcipt
scene.load.script('youtube', 'https://www.youtube.com/iframe_api');
```
which is not worked. Error messages
```
Access to XMLHttpRequest at 'https://www.youtube.com/iframe_api' from origin 'https://s.codepen.io' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

I can copy script file from `https://www.youtube.com/iframe_api` into my local site, to avoid this issue.

---
It is not a serious problem, please put this issue as lower priority.

**Comment by photonstorm** [comment]
Heya - not sure how Phaser can do anything else in this situation if CORs is blocking loading the script via xhr? There's no getting around that beyond just creating the script tag dynamically and settings its `src` attribute (i.e. not preloading the script at all)

**Comment by rexrainbow** [comment]
Thanks for this answer.
